### ISSUE TITLE
v2.0.17-beta TWO bugs

### ISSUE THREAD

**Issue by Ash1One** [CAMI-challenge/AMBER#35]
hello, @fernandomeyer 
I have met two bugs when using AMBER v2.0.17-beta.

1. index.html
![批注 2019-06-28 162303](https://user-images.githubusercontent.com/44107304/60328558-10bbed00-99c1-11e9-84fe-e2c45d899aab.jpg)
the pictures of `worst`, `medium`, `best` have failed to load.

2. purity_completeness_seq.png
![批注 2019-06-28 162702](https://user-images.githubusercontent.com/44107304/60328815-a35c8c00-99c1-11e9-8ae3-21900114b6ef.jpg)
purity_completeness_seq.png is blank.

**Comment by Ash1One** [comment]
and I have another question that if the parameter `-k` can remove two or more elements.
when I used `-k "circular element, unique element" `, it failed.
I will try `-k "circular element" -k "unique element" `, hope it work. 😀

**Comment by fernandomeyer** [comment]
Thank you for your feedback.

1. The broken figure is probably related to the version of matplotlib. Please update to AMBER v2.0.20-beta and use the Python packages listed in [requirements/default.txt.](https://github.com/CAMI-challenge/AMBER/blob/master/requirements/default.txt)
2. The blank purity vs. completeness plot does not look like a real issue. Please check if the value for these metrics is larger than zero.

Please reopen this issue if the problems persist.

**Comment by Ash1One** [comment]
Hi, @fernandomeyer  Thanks for reply.
I have just tried AMBER v2.0.20-beta, and conda env list:
```
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
biopython                 1.73                     pypi_0    pypi
bokeh                     0.13.0                   pypi_0    pypi
bzip2                     1.0.6             h14c3975_1002    conda-forge
ca-certificates           2019.6.16            hecc5488_0    conda-forge
cami-amber                2.0.20b0                 pypi_0    pypi
certifi                   2019.6.16                py37_0    conda-forge
cycler                    0.10.0                   pypi_0    pypi
jinja2                    2.10.1                   pypi_0    pypi
kiwisolver                1.1.0                    pypi_0    pypi
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 9.1.0                hdf63c60_0    defaults
libstdcxx-ng              9.1.0                hdf63c60_0    defaults
markupsafe                1.1.1                    pypi_0    pypi
matplotlib                3.1.1                    pypi_0    pypi
ncurses                   6.1               hf484d3e_1002    conda-forge
numpy                     1.16.4                   pypi_0    pypi
openssl                   1.1.1c               h516909a_0    conda-forge
packaging                 19.0                     pypi_0    pypi
pandas                    0.24.2                   pypi_0    pypi
pip                       19.1.1                   py37_0    conda-forge
pyparsing                 2.4.0                    pypi_0    pypi
python                    3.7.3                h33d41f4_1    conda-forge
python-dateutil           2.8.0                    pypi_0    pypi
pytz                      2019.1                   pypi_0    pypi
readline                  8.0                  hf8c457e_0    conda-forge
scipy                     1.3.0                    pypi_0    pypi
seaborn                   0.9.0                    pypi_0    pypi
setuptools                41.0.1                   py37_0    conda-forge
six                       1.12.0                   pypi_0    pypi
sqlite                    3.29.0               hcee41ef_0    conda-forge
tk                        8.6.9             hed695b0_1002    conda-forge
tornado                   6.0.3                    pypi_0    pypi
wheel                     0.33.4                   py37_0    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
zlib                      1.2.11            h516909a_1005    conda-forge
``` 
The broken figure still exist.
![20190715200624](https://user-images.githubusercontent.com/44107304/61214999-1133e200-a73c-11e9-9877-8b606a479ece.png)

**Comment by fernandomeyer** [comment]
Thanks again. I did find an issue with the plot purity_completeness_seq (Purity (seq) vs. Completeness (seq)) and fixed it in a new release 2.0.21-beta.

However, I could not reproduce the broken figure. Your conda env looks fine. Do you see figure heatmap_bar.png in the output directory of AMBER, in the same directory as index.html?

The following commands work for me.

`git clone https://github.com/CAMI-challenge/AMBER.git`

`cd AMBER`

`docker build -t amber:latest .`

`docker run --rm -v `pwd`:/host amber:latest 
amber.py 
-l "CONCOCT (CAMI), MaxBin 2.0.2 (CAMI)" 
-p 1 
-r /host/test/unique_common.tsv 
-k "circular element" 
-g /host/test/gsa_mapping.binning 
/host/test/goofy_hypatia_2 
/host/test/naughty_carson_2 
-o /host/output_dir`

**Comment by Ash1One** [comment]
@fernandomeyer Thank you.
I believe the broken figure is unrelated to AMBER as the problem only happend when I open the index.html from jupyter notebook. When downloading index.html and heatmap_bar.png to my PC, everything looks good.
